src/scripts/settings.ts

```diff
@@ -127,7 +128,7 @@ const settingsModule = {
   settingsWindow,
 };
 
-export const createSettingsWindow = function () {
+export const createSettingsWindow = function() {
```

Comment:
please revert these style changes so that they fit with the prettier config :)

Reply:
I think for some reason my formatter got confused with these functions in particular, changed them back.